generate_tarballs.sh

```diff
@@ -1,11 +1,17 @@
-#!/bin/bash -e
+#!/usr/bin/env bash
```

Comment:
You need "-e" here, keep compatibility.

Reply:
Adding `-e` to `#!/usr/bin/env bash` breaks everything.  AFAIK using `#!/usr/bin/env bash` [is recommended].  I have no idea what `-e` does, I'm happy to go with anything you recommend :).

[is recommended]: https://stackoverflow.com/a/10383546/390493

Reply:
I do not object with `#!/usr/bin/env bash`.
It is certainly a good practice

* In a case of people want to run a bash script on the environment where `/bin/bash` does not exist. But I wonder if there is the Linux or UNIX distribution that do not have it today. Because `/bin/` is standard rule of Linux system. Ref: https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard . 
* I a case of people want to run a shell script on the different path's bash that is source compiled bash.

The down side of `#!/usr/bin/env bash` is different path's bash is unintentionally executed. 

I am not a original coder who wrote `#!/bin/bash`. But you should feel original coder's good intent for the code. At least you should not include the change of `#!/usr/bin/env bash` in this PR. Because it's not harmful.
For each change especially for non backward compatibility feature, we need discussion.

Reply:
> I have no idea what -e does, I'm happy to go with anything you recommend :).

You do not know `bash -e`? It's quite important feature to make the bash script defensive. It's similar with `set -e`.  `man bash` might explain you.

When you have no idea what something does, proceeding removing it is not good choice.